documentation/decision-service-framework.asciidoc

```diff
@@ -3,40 +3,13 @@ toc::[]
+We need to choose which framework(s) we want to use for building services. For the devonfw, we focus on a standard API, if available. However, we also want to recommend an implementation. While projects would still be able to choose whatever they want, we want to suggest the best, most robust, and established solution. This way, projects do not have to worry about the decision and can rely on a production-ready framework without running into any trouble. Also, besides the standard, the configuration of the implementation framework differs, so we want to give instructions  in the documentation and by our sample application. This is why, in the end, the implementation also matters. If a project has a customer demand to use something else, the project has to take care of it. We will always suggest and "support" ONE solution.
 
 == REST Services
-For REST services we rely on the JAX-RS standard (and NOT on spring-mvc with its proprietary annotations).
-As JAX-RS implementation had Jersey (Reference-Implementation) and Apache CXF on the short-list. As it turned out, Jersey is build for HK2 and the spring integration is rather a quickfix. So we ran into bugs when we used it with spring. There are various issues open in the jersey tracker related to this. E.g. https://java.net/jira/browse/JERSEY-2112
-For Apache CXF the spring container was first choice but container abstraction has been properly introduced by design so it can be used in JEE application servers. Everything works smooth in our sample application and we collected feedback from various projects doing JAX-RS based on CXF either with XML or JSON with success in production. 
+For REST services, devonfw relies on the JAX-RS standard (and NOT on spring-mvc with its proprietary annotations). https://github.com/eclipse-ee4j/jaxrs-api[JAX-RS] (Jakarta RESTful Web Services) is a Java programming language API to develop web services following the Representational State Transfer (REST) architectural pattern. 
+For https://cxf.apache.org/[Apache CXF], the spring container was the first choice, but container abstraction has been properly introduced by design, so it can be used in JEE application servers. Apache CXF is a services framework that helps to build and develop services using frontend programming APIs, such as JAX-RS. Everything works smoothly in our sample application, and in addition, we collected feedback from various projects utilizing CXF, either with XML or JSON, with reported success in production. Therefore, we decided to use Apache CXF for Spring.
+For Quarkus applications, devon4j recommends to use RESTEasy. RESTEasy is a JAX-RS implementation aimed at providing productivity frameworks for developing client and server RESTful applications and services in Java (https://github.com/resteasy/resteasy)
```

Comment:
like [Julian mentioned here](https://github.com/devonfw/devon4j/pull/506#discussion_r759941442) also change the RESTeasy link.